docs/setup/independent/install-kubeadm.md

```diff
@@ -12,13 +12,14 @@ This page shows how to use install kubeadm.
 
 * One or more machines running Ubuntu 16.04+, Debian 9, CentOS 7, RHEL 7, Fedora 25/26 (best-effort) or HypriotOS v1.0.1+
 * 1GB or more of RAM per machine (any less will leave little room for your apps)
+* 2CPU or more 
 * Full network connectivity between all machines in the cluster (public or private network is fine)
 * Unique MAC address and product_uuid for every node
 * Certain ports are open on your machines. See the section below for more details
 * Swap disabled. You must disable swap in order for the kubelet to work properly.
-* Set `/proc/sys/net/bridge/bridge-nf-call-iptables` to `1` by running `sysctl net.bridge.bridge-nf-call-iptables=1`
-to pass bridged IPv4 traffic to iptables' chains. This is a requirement for CNI plugins to work, for more information
-please see [here](https://kubernetes.io/docs/concepts/cluster-administration/network-plugins/#network-plugin-requirements).
+* If you don't plan to use Calico or Canal as your networking plugin, set `/proc/sys/net/bridge/bridge-nf-call-iptables` to `1` by running `sysctl net.bridge.bridge-nf-call-iptables=1`
```

Comment:
Do you mean that Calico & Canal _do not work_ with this setting?
I think this should be documented in the network plugin tabs

Reply:
I am not sure how you want to word this, but the way it is worded at present is incorrect. Calico and Canal do _not_ require `sysctl net.bridge.bridge-nf-call-iptables=1`. They will ignore this setting. cc: @caseydavenport

Reply:
Yeah, calico / canal don't use a linux bridge and so this setting has no effect.

@luxas I agree - this feels like a per-network bit of documentation. Perhaps we should move this to the network tabs for those options which require it.

Reply:
I think this should be provider-specific in the tabs